### DISCUSSION TITLE
有输出更简洁的html内容 以及放弃在每个组件中定义大量css变量和style内容的改进计划吗?

### DISCUSSION THREAD

**Discussion by coader** [tusen-ai/naive-ui#3347] [General]
现在这样的输出虽然精细度是够了 但实际上几乎用不着这么精细
虽然暂时未发现性能问题 可能是组件数不够多
不知是否有新的版本计划？

**Comment by 07akioni** [comment]
发现性能问题可以报，用直觉来推断其实是很容易出问题的

**Reply by 07akioni** [reply]
如果不想看 inline 样式可以在 config provider 配置 inline-theme-disabled

性能基本没有变化

**Comment by coader** [comment]
尝试了一下 radiobutton的cursor pointer被忽略了

![image](https://user-images.githubusercontent.com/3700498/180999439-454163f4-b4e4-4b66-b95d-1ba42528a8fc.png)

**Reply by 07akioni** [reply]
老版本有过这个 bug？我没复现，在现在的版本

<img width="1195" alt="image" src="https://user-images.githubusercontent.com/18677354/181056889-b705f3e4-c6dd-4fa8-85a4-d5abb3871f16.png">

**Comment by coader** [comment]
查了半天 包括刚升级最新 "naive-ui": "^2.32.0" 然后发现我的
![image](https://user-images.githubusercontent.com/3700498/181675845-2f57b3dd-688c-4633-a40a-3d3496a503bb.png)
少了pointer-events设置 这已经是去除了那个 inline-theme-disabled 的恢复原设置下的

![image](https://user-images.githubusercontent.com/3700498/181675941-4b06bd63-bdd3-45db-8462-563e7a21bf74.png)

**Comment by coader** [comment]
用pnpm install --force强制重新安装了仍然如此

**Comment by coader** [comment]
彻底删除干净重新安装后好了 估计还是之前某个版本的问题
